interface/main/messages/messages.php

```diff
@@ -49,6 +49,8 @@
         exit();
     }
     $logged_in = $MedEx->login();
+} else {
+    $logged_in = '';
```

Comment:
may be better to make this null in case something just is checking to see if it is set (considering this appears to be checking for some sort of auth)